domains/transfer-agreements/TransferAgreements.API/API.UnitTests/Data/TransferAgreementServiceTests.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Threading.Tasks;
+using API.Data;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using Xunit;
+
+namespace API.UnitTests.Data
+{
+    public class TransferAgreementServiceTests
+    {
+        [Fact]
+        public async Task CreateTransferAgreement_ShouldAddTransferAgreementToDatabase()
+        {
+
+            var options = new DbContextOptionsBuilder<ApplicationDbContext>()
+                .UseInMemoryDatabase(databaseName: "TestDatabase")
+                .Options;
+
+            using (var context = new ApplicationDbContext(options))
```

Comment:
You can inline this using statement.
I am not sure if this is what we want, this is spinning a memory db up. We are doing the same thing in the integration test.
Is it possible to simple mock the DB?